src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -38,4 +45,68 @@ template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
+///
+/// Can be retrieved using `gr::Tags::SpatialChristoffelFirstKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct SpatialChristoffelFirstKindCompute
+    : SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags = tmpl::list<
+      ::Tags::deriv<gr::Tags::SpatialMetric<SpatialDim, Frame, DataType>,
+                    tmpl::size_t<SpatialDim>, Frame>>;
+  static constexpr tnsr::ijj<DataType, SpatialDim, Frame> (*function)(
+      const tnsr::ijj<DataType, SpatialDim, Frame>&) =
+      &christoffel_first_kind<SpatialDim, Frame, IndexType::Spatial, DataType>;
+  using base = SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>;
+  using type = typename base::type;
+};
+
+/// Compute item for spatial Christoffel symbols of the second kind
+/// \f$\Gamma^i_{jk}\f$ computed from the Christoffel symbols of the
+/// first kind and the inverse spatial metric.
+///
+/// Can be retrieved using `gr::Tags::SpatialChristoffelSecondKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct SpatialChristoffelSecondKindCompute
+    : SpatialChristoffelSecondKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags =
+      tmpl::list<SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>,
+                 InverseSpatialMetric<SpatialDim, Frame, DataType>>;
+  static constexpr tnsr::Ijj<DataType, SpatialDim, Frame> (*function)(
+      const tnsr::ijj<DataType, SpatialDim, Frame>&,
+      const tnsr::II<DataType, SpatialDim, Frame>&) =
+      &raise_or_lower_first_index<DataType,
+                                  SpatialIndex<SpatialDim, UpLo::Lo, Frame>,
+                                  SpatialIndex<SpatialDim, UpLo::Lo, Frame>>;
+  using base = SpatialChristoffelSecondKind<SpatialDim, Frame, DataType>;
+  using type = typename base::type;
+};
+
+/// Compute item for the trace of the spatial Christoffel symbols
+/// of the first kind
+/// \f$\Gamma_{i} = \Gamma_{ijk}g^{jk}\f$ computed from the
+/// Christoffel symbols of the first kind and the inverse spatial metric.
+///
+/// Can be retrieved using `gr::Tags::TraceSpatialChristoffelFirstKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct TraceSpatialChristoffelFirstKindCompute
+    : TraceSpatialChristoffelFirstKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags =
+      tmpl::list<SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>,
+                 InverseSpatialMetric<SpatialDim, Frame, DataType>>;
+  static constexpr tnsr::i<DataType, SpatialDim, Frame> (*function)(
+      const tnsr::ijj<DataType, SpatialDim, Frame>&,
+      const tnsr::II<DataType, SpatialDim, Frame>&) =
+      &trace_last_indices<DataType, SpatialIndex<SpatialDim, UpLo::Lo, Frame>,
+                          SpatialIndex<SpatialDim, UpLo::Lo, Frame>>;
+  using base = TraceSpatialChristoffelFirstKind<SpatialDim, Frame, DataType>;
```

Comment:
`base` and `type` are not part of the compute item interface. Please remove these.

Reply:
see [here](https://github.com/sxs-collaboration/spectre/pull/1478#discussion_r282530726)